app/src/openxr/cpp/DeviceDelegateOpenXR.cpp

```diff
@@ -248,6 +264,15 @@ struct DeviceDelegateOpenXR::State {
         else
             passthroughStrategy = std::make_unique<OpenXRPassthroughStrategyUnsupported>();
     }
+
+    if (handMeshPropertiesMSFT.supportsHandTrackingMesh) {
+        handMeshMSFT.isSupported = true;
+        handMeshMSFT.vertexCount = handMeshPropertiesMSFT.maxHandMeshVertexCount;
+        handMeshMSFT.indexCount = handMeshPropertiesMSFT.maxHandMeshIndexCount;
+        VRB_LOG("OpenXR runtime supports XR_MSFT_hand_tracking_mesh");
```

Comment:
Ditto.

Reply:
This one was not a left over. The intention was to log whether the runtime does support `XR_MSFT_hand_tracking_mesh` extension, similar to the log we have for `XR_EXT_hand_tracking`.

Reply:
ok